event_model/schemas/bulk_datum.json

```diff
@@ -0,0 +1,27 @@
+{
+    "properties": {
+        "datum_kwargs_list": {
```

Comment:
It is `datum_kwarg_list` https://github.com/NSLS-II/databroker/blob/bdfef404ede4eb94f78dec8769ac3d7c675b4484/databroker/assets/base_registry.py#L639-L643